applications/StructuralMechanicsApplication/structural_mechanics_application.h

```diff
@@ -309,6 +309,8 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) KratosStructuralMechanicsAppl
     const SmallDisplacement mSmallDisplacement2D6N;
     const SmallDisplacement mSmallDisplacement2D8N;
     const SmallDisplacement mSmallDisplacement2D9N;
+    const SmallDisplacement mSmallDisplacement2D10N;
```

Comment:
It would be interesting to extend to TL and UL elements

Reply:
I totally agree with @loumalouomega , it could be totally useful for the community

Reply:
Thank you @loumalouomega and @AlejandroCornejo 
I already added 4 elemnts:

TotalLagrangianElement2D10N
TotalLagrangianElement2D15N
UpdatedLagrangianElement2D10N
UpdatedLagrangianElement2D15N

Will be in the coming commit (together with some other fixes)